test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SelectNewLeadControllerSpec.scala

```diff
@@ -33,14 +39,33 @@ class SelectNewLeadControllerSpec
   extends ControllerSpec
     with AuthSupport
     with JourneyStoreSupport
-    with AuthAndSessionDataBehaviour {
+    with AuthAndSessionDataBehaviour
+    with RetrieveEmailSupport
+    with SendEmailSupport {
 
   val mockEscService = mock[EscService]
 
   override def overrideBindings           = List(
     bind[AuthConnector].toInstance(mockAuthConnector),
     bind[Store].toInstance(mockJourneyStore),
-    bind[EscService].toInstance(mockEscService)
+    bind[EscService].toInstance(mockEscService),
+    bind[RetrieveEmailService].toInstance(mockRetrieveEmailService),
+    bind[SendEmailService].toInstance(mockSendEmailService)
+  )
+
+  override def additionalConfig = super.additionalConfig.withFallback(
+    Configuration(
+      ConfigFactory.parseString(s"""
+                                   |
+                                   |play.i18n.langs = ["en", "cy", "fr"]
```

Comment:
Minor - You can use something like 

```
    Configuration.from(Map(
      "play.i18n.langs" -> Seq("en", "cy", "fr"),
      "email-send.create-undertaking-template-en" -> "template_EN",
      "email-send.create-undertaking-template-cy" -> "template_CY",
    )))
```

instead of parsing a string.

Reply:
I think both are same . I prefer parse string as it shows how the actual config is and how we are reading from it.  Sometimes it helps in spotting a bug in actual config file. What do you think?

Reply:
I don't feel strongly either way. I tend to go for maps here because I'd rather not have the runtime string parsing. Personal preference I guess! :)